### DISCUSSION TITLE
请教大佬么，如何让提醒脚本短时间内只提醒一次？

### DISCUSSION THREAD

**Discussion by Cyiming** [whyour/qinglong#2223] [Q&A]
我写了个库存查询脚本 1分钟一次 查到库存通过pushplus给我发送提醒 但有时候一段时间都有库存会造成每分钟一次提醒 pushplus会封号 怎么做到查到库存给我提醒后 半个小时后再查询呢 而不是默认一分钟

**Comment by TMM0011** [comment]
解决了吗兄弟，脚本能分享吗

**Reply by Cyiming** [reply]
> 解决了吗兄弟，脚本能分享吗

没有

**Comment by pooneyy** [comment]
改写脚本本身，在脚本内部循环查询，而不是频繁运行脚本

**Reply by Cyiming** [reply]
> 改写脚本本身，在脚本内部循环查询，而不是频繁运行脚本

不现实啊 除非脚本让脚本死循环一直运行 其实我想到方案了 就是把检索频率放到环境变量中 然后修改环境变量 但我不会
